### PR TITLE
Makefile: do not depend on TARGET for install-bash-completion

### PR DESCRIPTION
**Pull request by aanm**
As it was done before 6fb045dc3dac ("Makefile: add install-bash-completion target")
the install target did not depend on TARGET. This fixes up the change to
have the same behavior as before.

Fixes: 6fb045dc3dac ("Makefile: add install-bash-completion target")
Signed-off-by: André Martins <andre@cilium.io>

### CODE REVIEW

**Comment by aanm** [comment]
test-me-please

**Review by twpayne — APPROVED** [review]
LGTM. I would reword the commit message to be

    Makefile: add install-bash-completion-only target

as after this change, the `install-bash-completion` still depends on TARGET.
